app/images/appleAppStoreButton/da.svg

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
FWIW, we may want to use lowercase folders w/ underscores, for consistency.
For example, _"app/images/apple_app_store_button/**{{locale}}**.svg"_ and _"app/images/google_play_store_button/**{{locale}}**.png"_.